content/posts/cjd-casc.md

```diff
@@ -0,0 +1,548 @@
+---
+author:
+  name: "Matt Elgin"
+title: "Self-Updating Jenkins: GitOps for Jenkins Configuration"
+date: 2019-07-01T17:00:00-04:00
```

Comment:
Update publish date and change `draft` to false once you have reviewed recommended changes. After that, it looks good to me! Great work @mdelgin!